photutils/morphology.py

```diff
@@ -5,14 +5,27 @@
 """
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
+import collections
 import numpy as np
-from astropy.modeling import models
+from astropy.modeling.models import Gaussian1D, Gaussian2D, Const1D, Const2D
 from astropy.modeling.fitting import LevMarLSQFitter
+from astropy.nddata.utils import overlap_slices
 from .segmentation import SegmentProperties
+import warnings
 
 
-__all__ = ['centroid_com', 'gaussian1d_moments', 'centroid_1dg',
-           'centroid_2dg', 'fit_2dgaussian', 'data_properties']
+__all__ = ['GaussianConst1D', 'GaussianConst2D', 'centroid_com',
```

Comment:
Now it gets interesting ... we need lots of compound models in Gammapy ... is it possible to make them well-documented or is the simplest way to just sub-class `Model` and define it from scratch by calling the component `evaluate` static methods in the compound model `evaluate` method?